### ISSUE TITLE
Rename ethnicity and ethnicity_ontology_term_id 

### ISSUE THREAD

**Issue by jahilton** [chanzuckerberg/single-cell-curation#199]
There are several issues arising from attempts to standardize donor ethnicity

1. The current schema standardizes using HANCESTRO, an ontology designed to capture _ancestry_, not ethnicity. So not the intended purpose of the ontology, plus these are two concepts that can be confused for each other by researchers, so entangling them is not ideal.
2. The recorded values are typically 1) self-reported and 2) limited to the options put in front of a patient. Without standardization of those options, standardization of the recorded values is incredibly difficult.
3. A majority of primary human data in cellxgene currently is 'unknown' for ethnicity (quick sum of datasets that are human and all cells are is_primary_data=True - datasets that are all 'unknown' total 8,384,599 cells while the rest total 5,920,981 cells
5. Ethnicity is a complex concept. We recently had the following cases reported for patients in a Seed Network study (each bullet is the record ethnicity for a single patient)...
- White and Asian, Japanese
- White and Ashkenazi Jew
- Black and Ashkenazi Jew
- Black and white, Northwestern European
- Black, Legacy African-american and white, Western Asian and North African
- Afro-Caribbean and Hispanic or Latin American

**Comment by jahilton** [comment]
Some other groups that would be valuable to hear from on this topic (not necessarily schema)

- [HCA Genetic Diversity WG](https://www.humancellatlas.org/biological-networks/) - I have joined
- [NAS Committee on Use of Race, Ethnicity, and Ancestry as Population Descriptors in Genomics Research](https://www.nationalacademies.org/our-work/use-of-race-ethnicity-and-ancestry-as-population-descriptors-in-genomics-research) - upcoming workshop on Apr 4
- [ClinGen Ancestry and Diversity Forum](https://clinicalgenome.org/tools/ancestry-and-diversity-forum/) - I have joined & they have their previous presentations all recorded and available on the website

**Comment by jahilton** [comment]
I had a great conversation with Alice Popejoy (co-chair of the ClinGen Ancestry and Diversity WG, which runs the Forum mentioned above & author of [this informative piece](https://www.nature.com/articles/d41586-021-02288-x) that outlines issues with the terms & concepts used in genomics research).
A few notes:

- capturing self-reported ethnicity at the cellular level which we do is going to make it incredibly likely that researchers will treat it as a biologically-connected concept, which it is not.
- in doing so, they may find correlation, and mistakenly assign causation to ethnicity (when, in fact, ethnicity may be a proxy for geography, diet, any number of environmental variables, batch effects, etc. in a given study and those are the significant variables).
-  there's a risk that in addition to that mis-assigned causation, they then take those findings, draw conclusions, and those reinforce biological racism that's already prevalent due to these rampant assumptions made by others previously

If the only reason to capture ethnicity is to gauge the diversity of populations represented in the data corpus, then this could be ingested as it is now, but removed from the cxg explorer & downloadable datasets. Then used to summarize the whole data corpus.
If that's not even a good enough reason, then we just remove it from the schema all together.

**Comment by jahilton** [comment]
**Proposal from Lattice**
- Remove `ethnicity_ontology_term_id` and `ethnicity` from the schema.
- Add `self_reported_ethnicity` to the schema, a free text field (`unknown` is still used to capture the lack of information). The `self_reported_` can help further disambiguate the information from genotyped ancestry. Curators can do our best to ensure capitalization is consistent, typos do not appear, delimiters are consistent, etc.
- Remove the ethnicity filter on Dataset/Collection search pages. This is likely not a common filtering use case anyway, but instead information to consider later when evaluating the variability/diversity in an already-filtered dataset.
- To migrate, remove `ethnicity_ontology_term_id` from existing datasets. `ethnicity` can remain, and Lattice can, over time, update those standardized labels to the originally reported values as we can.

The proposal is a statement that the information is valuable, but we don't know how to standardize yet. So we will collect it as-is for now and standardize later on.

Continuing the use of HANCESTRO terms as an enum is going to result in more information loss. Reported values are being converted to a set of values that we know to be insufficient and even misleading (the _ancestry_ ontology is used to capture _ethnicity_). In some cases, multiple values are being reported and they're being condensed into one. The original reported values are not being retained.
Adopting a new enum risks the same information loss, even if we allow for multiple values to be selected.

Removing this field from the schema altogether (thus, making it optional) will result in even less ethnicity information submitted.

**Comment by brianraymor** [comment]
Reviewed with **small group** (Norbert, Jason, Harley, Kuni, Brian R).

TLDR - Agreed to adopt [Lattice proposal](https://github.com/chanzuckerberg/single-cell-curation/issues/199#issuecomment-1149268863) despite misgivings about the value of _free text_ to inform a future controlled vocabulary or ontology. 

> To migrate, remove ethnicity_ontology_term_id from existing datasets. ethnicity can remain, and Lattice can, over time, update those standardized labels to the originally reported values as we can.

@jahilton - to help with this migration do you require the curator API to continue returning the ethnicity `label` for legacy datasets in responses or may both the `label` and `ontology_term_id` be removed from the portal database and responses? If required, then my recommendation would be to simply return the current response unchanged (perhaps with an annotation of "deprecated" to warn others):

```
        "ethnicity": [
          {
            "label": "Asian",
            "ontology_term_id": "HANCESTRO:0008"
          },
```

#### Additional requirements

Lattice is responsible for manual validation of `self_reported_ethnicity` including separators. The `cellxgene-schema` CLI MUST ignore. 

`self_reported_ethnicity` MUST appear under **Author Categories** in the cellxgene explorer LSB and NOT **Standard Categories**. 

**Ethnicity** MUST be removed from analytics dashboards.

**Comment by ainfeld** [comment]
@brianraymor thanks for the visibility! 

Do you want to replace current "Ethnicity" with what is in the `self_reported_ethnicity` field or remove any analytics on the entire topic? 

I assume that means the ethnicity filter will be removed as well?

**Comment by brianraymor** [comment]
> Do you want to replace current "Ethnicity" with what is in the self_reported_ethnicity field or remove any analytics on the entire topic?

Please remove until we can assess the set of values being returned by `self_reported_ethnicity`.


> I assume that means the ethnicity filter will be removed as well?

Yes.

**Comment by ainfeld** [comment]
I've removed "Ethnicity" from the `Cellxgene - data availability` dashboard

Once the filter is removed from the UI I will remove that part from the `Cellxgene - usage` dashboard

**Comment by brianraymor** [comment]
> Once the filter is removed from the UI I will remove that part from the Cellxgene - usage dashboard

@ainfeld - I will be creating an epic in single-cell to model and track all the required changes related to the deprecation. Do you have a repo where you track dashboard issues? If so, I could create an issue and add a blocking dependency on the filter removal.

**Comment by ainfeld** [comment]
@brianraymor I'm just in the process of setting up a system to track requests for changes to the dashboard. I just created a #single-cell-analytics repo to link my notebooks in databricks with the scripts to extract data for the dashboards so that the engineers can do fairly traditional PR requests. Would it work to create an issue in that repo?

**Comment by brianraymor** [comment]
Yes. I've added that repo to our zenhub workspace.

**Comment by signechambers1** [comment]
@brianraymor we will also want to remove the ethnicity filter on scExpression as part of the deprecation. Once you create the deprecation epic let me know and I can link an issue for scExpression. Thank you!

**Comment by brianraymor** [comment]
@jahilton 

RE

> To migrate, remove ethnicity_ontology_term_id from existing datasets. ethnicity can remain, and Lattice can, over time, update those standardized labels to the originally reported values as we can.

Would it not be more consistent for Lattice to delete `ethnicity_ontology_term_id` and rename `ethnicity` to `self_reported_ethnicity` immediately. In general, we want to start warning/failing when previous metadata exists in datasets.

**Comment by jahilton** [comment]
If we are planning to warn/fail on any previously-used metadata fields, then I would expect the schema to explicitly state which fields are not allowed. Is this the plan?

The renaming solution would actually be inconsistent with the schema for `self_reported_ethnicity` which will be advertised as free text when those migrated values would still be "bent" to HANCESTRO.

**Comment by ambrosejcarr** [comment]
> The renaming solution would actually be inconsistent with the schema for self_reported_ethnicity which will be advertised as free text when those migrated values would still be "bent" to HANCESTRO.

@jahilton I wanted to verify: completely absent is better than "bent to HANCESTRO"?

**Comment by jahilton** [comment]
>completely absent is better than "bent to HANCESTRO"?

No, knowing now that `ethnicity` will be disallowed, then migration should be renaming the existing `ethnicity` values to `self_reported_ethnicity` immediately.

**Comment by brianraymor** [comment]
@jahilton - My aspiration is to have a `cellxgene-schema` CLI option to automate some of the 2.0->3.0 migration for Lattice:

1. Delete `ethnicity_term_ontology_id`
1. Rename `ethnicity` to `self_reported_ethnicity`
1. Delete `X_normalization`
1. Update `schema_version` to "3.0.0"
1. Remove human-readable labels for ontology term id(s)
1. etc.

**Comment by jahilton** [comment]
@brianraymor I always liked you

**Comment by brianraymor** [comment]
Closing now that we're in alignment.

**Comment by brianraymor** [comment]
Re-opened. Per July 12 call with Norbert, @jahilton, and @hthomas-czi - we agreed to rename and reuse the current ethnicity field with its HANCESTRO value. 

### self_reported_ethnicity_ontology_term_id

<table><tbody>
    <tr>
      <th>Key</th>
      <td>self_reported_ethnicity_ontology_term_id</td>
    </tr>
    <tr>
      <th>Annotator</th>
      <td>Curator</td>
    </tr>
    <tr>
      <th>Value</th>
        <td>categorical with <code>str</code> categories. If <code>organism_ontolology_term_id</code> is <code>"NCBITaxon:9606"</code> for <i>Homo sapiens</i>, this MUST be either a HANCESTRO term or <code>"unknown"</code> if unavailable. <br><br>Otherwise, for all other organisms this MUST be <code>"na"</code>.
        </td>
    </tr>
</tbody></table>
<br>

### self_reported_ethnicity

<table><tbody>
    <tr>
      <th>Key</th>
      <td>self_reported_ethnicity</td>
    </tr>
    <tr>
      <th>Annotator</th>
      <td>Data Portal</td>
    </tr>
    <tr>
      <th>Value</th>
        <td>categorical with <code>str</code> categories. This MUST be <code>"na"</code> or <code>"unknown"</code> if set in <code>self_reported_ethnicity_ontology_term_id</code>; otherwise, this MUST be the human-readable name assigned to the value of <code>self_reported_ethnicity_ontology_term_id</code>.
        </td>
    </tr>
</tbody></table>
<br>

I've proposed that the pipeline 3.0.0 re-validation process automate the rename on behalf of the curators.

**Comment by brianraymor** [comment]
Addressed in #268
